src/lib/undux/utils/feed.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import type { Effects, Store } from 'undux'
+
+import userStorage from '../../gundb/UserStorage'
+import pino from '../../logger/pino-logger'
+const logger = pino.child({ from: 'feeds' })
+
+const PAGE_SIZE = 10
```

Comment:
this value has any relation to the `initialNumToRender={10}` in the `FeedList.js`? If they have to be the same, we can use the same const.

Reply:
It's not necessarily the same since in the list is how many should be draw in screen and PAGE_SIZE is how many will get from gundb to undux but it makes sense in this case to use same constant I think. Just made this change